tests/containers/docker_compose.pm

```diff
@@ -53,7 +51,6 @@ sub run {
     assert_script_run("curl -O " . data_url("containers/docker-compose.yml"));
     assert_script_run("curl -O " . data_url("containers/haproxy.cfg"));
 
-    $docker->configure_insecure_registries();
```

Comment:
in this certain case it become more severe because we changing order of execution . are we sure that it won't influence on what we actually testing ?

Reply:
it doesnt seem to make any difference. i would say it is completely save IMO

Reply:
I also gave it a thought when I removed it. I don't know what it was placed here instead of the beginning, I guess because of convenience, you add it when you needed it. This shouldn't affect the flow of the test.